adobe/react-spectrum
# fix(#2592): Breadcrumbs lose focus when a breadcrumb link is clicked to become the current page

Closes #2592

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 2592](https://github.com/adobe/react-spectrum/issues/2592).

## 📝 Test Instructions:

1. Open Storybook story:  https://reactspectrum.blob.core.windows.net/reactspectrum/f4ecf8c4cfa077f178b9fb6d6e465291dcb2193c/storybook/index.html?path=/story/breadcrumbs--dynamic-breadcrumbs-with-focus-management
2. Move keyboard focus to one of the links preceding the current location.
3. Activate the link to update the current location.
4. Focus should get set to the new current location, which is a "disabled" link with `aria-current` and text matching the text for the link you clicked.
5. You can reset the breadcrumbs to how they were at the start using the "Reset" button.
5. Repeat for items within the dropdown menu.

## 🧢 Your Project:

Adobe/Accessibility
